cni/log/logger.go

```diff
@@ -1,55 +1,43 @@
 package log
 
 import (
-	"context"
-	"fmt"
-	"os"
-
-	"go.uber.org/zap"
+	"github.com/Azure/azure-container-networking/log"
+	"github.com/Azure/azure-container-networking/zaplog"
 	"go.uber.org/zap/zapcore"
-	"gopkg.in/natefinch/lumberjack.v2"
 )
 
-type Config struct {
-	Level       zapcore.Level
-	LogPath     string
-	MaxSizeInMB int
-	MaxBackups  int
-	Name        string
-	Component   string
-}
-
-var Logger *zap.Logger
+const (
+	maxLogFileSizeInMb = 5
+	maxLogFileCount    = 8
+)
 
-// Initializes a Zap logger and returns a cleanup function so logger can be cleaned up from caller
-func Initialize(ctx context.Context, cfg *Config) {
-	Logger = newFileLogger(cfg)
+var (
+	LoggerVnetName      string
+	LoggerIpamName      string
+	LoggerTelemetryName string
+	LogLevel            int8
+)
 
-	go func() {
-		<-ctx.Done()
-		err := Logger.Sync()
-		if err != nil {
-			fmt.Println("failed to sync logger")
-		}
-	}()
+var LoggerVnetCfg = &zaplog.Config{
+	Level:       zapcore.DebugLevel,
+	LogPath:     log.LogPath + "azure-vnet.log",
+	MaxSizeInMB: maxLogFileSizeInMb,
+	MaxBackups:  maxLogFileCount,
+	Name:        LoggerVnetName,
```

Comment:
whats the purpose of this field in logging?

Reply:
Hi Tamilmani, this name was used before, like:
```
{"level":"info","ts":"2023-08-25T13:48:27.199Z","msg":"Plugin Info","pid":1193999,"component":"cni",**"name":"azure-vnet"**,"version":"v1.5.10-11-gb0178db1"}
{"level":"info","ts":"2023-08-25T02:37:53.900Z","msg":"Plugin Info","pid":344266,"component":"cni",**"name":"azure-vnet"**,"version":"v1.5.10-9-ge789e04f"}
```